mmdet/core/anchor/atss_target.py

```diff
@@ -0,0 +1,170 @@
+import torch
```

Comment:
We prefer to put `atss_target`as a function of `atss_head` instead separating it as an independent file.
You can refer the code design of `fcos_head.py`